### PR TITLE
[FTML-93] Fix HTML rendering issues

### PR DESCRIPTION
**Pull request by emmiegit**
Follow-up to #188 	

In producing HTML tests based on AST test inputs, several issues were found. [FTML-93](https://scuttle.atlassian.net/browse/FTML-93) is a collective issue to resolve these. This does not include any changes to the collapsible, that will be a separate PR.

### CODE REVIEW

File: ftml/src/render/html/element/text.rs [hunk 1]
```diff
@@ -69,8 +61,10 @@ pub fn render_code(
 
     // TODO: syntax highlighting based on 'language'
 
-    ctx.html()
-        .div()
+    ctx.html() //
+        .pre()
         .attr("class", &["code"])
```
Comment (Monkatraz) [inline ftml/src/render/html/element/text.rs]: IIRC, the way it was "recommended" (HTML5 spec, I think?) to handle code blocks of a specific language type was to use a class, specifically `language-xxxx` or `lang-xxxx`. Should there be a default `language-none` (or something) class for unlabeled code blocks?

**Review by Monkatraz — APPROVED** [review]
Looks good to me.

File: ftml/src/render/html/element/text.rs (hunk 1 above)
Comment (emmiegit) [reply ftml/src/render/html/element/text.rs]: I considered that but didn't end up adding one. I'll add it now.

**Comment by emmiegit** [comment]
thanks @Monkatraz

**Pull request merged**